sources/install/package_wordlists.sh

```diff
@@ -63,6 +63,16 @@ function install_seclists() {
     add-to-list "seclists,https://github.com/danielmiessler/SecLists,A collection of multiple types of lists used during security assessments"
 }
 
+function install_onelistforall() {
+    colorecho "Installing onelistforall"
+    cd /opt/seclists/Discovery/Web-Content || exit
```

Comment:
Maybe it would be best to install unit wordlists in a custom directory? (e.g. `/opt/lists/`?)

Reply:
make me think that if we do this, we'll need to modify `fzf-wordlists` so that it look in that directory as well, and maybe have simlinks to other lists there

Reply:
what do you think @QU35T-code ?

Reply:
Yes, I agree with `/opt/lists`. We can also add john's rules in it ? `/opt/tools/john/run/rules`

Reply:
And also Seclists ?